.github/workflows/support.yml

```diff
@@ -11,16 +11,16 @@ jobs:
   action:
     runs-on: ubuntu-latest
     steps:
-      - uses: dessant/support-requests@v2
+      - uses: dessant/support-requests@v4
         with:
           github-token: ${{ github.token }}
           support-label: 'support'
           issue-comment: >
             👋 We use the issue tracker exclusively for bug reports and feature requests.
             However, this issue appears to be a support request. Please use our
-            [support channels](https://github.com/kivy/pyjnius/blob/master/README.md#support)
+            [support channels](https://github.com/kivy/kivy/blob/master/README.md#support)
```

Comment:
```suggestion
            [support channels](https://github.com/kivy/pyjnius/blob/master/README.md#support)
```

Reply:
First, I should redirect to the CONTACT.md file rather than the section in the readme.

But, to the bigger issue - The PyJNIus version just references the Kivy/Kivy version. I don't mind sending browsers along chains of "302 Redirects", but I didn't want to send humans to a page that just said "The princess is in another [castle](link).".

What do you think?

Reply:
It's a line call. Changed it.